script/Ciel/Day 14/14_01_CIEL14_1B.txt

```diff
@@ -21,7 +21,7 @@ Each and every building...
 [sha:2cacdfc9f05cc4d3282f49d750d7fc09be635951]{
 -- Page 10001, Offset 37140.
 -- 建物という建物にかかる、蜘蛛の糸のような繭。
-Each and every structure is cocooned in what looks like spider's silk.
+Each and every structure is cocooned in what looks like spider silk.
```

Comment:
First one sounds slightly nicer to me when I say it but can't elaborate why

Reply:
Just spider silk is more common but I guess we can make an exception if it sounds nicer
![image](https://user-images.githubusercontent.com/90473809/234771363-4f15dd8b-ce8d-41c4-903b-224199dd46e4.png)